openapi/src/components/entities/null.yml

```diff
@@ -0,0 +1,4 @@
+type: string
```

Comment:
This is a workaround and utility for nullable objects.

In OpanAPI 3.0, the nullable attribute is expressed as follows
```yml
properties:
  nullable_string:
    type: string
    nullable: true
```

In other hand, OpenAPI can reuse schema via `$ref`.

However, This definnition does not work as nullable attribute.
```yml
properties:
  nullable_object:
    nullable: true
    $ref: "./object.yml"
```

In this case, this workaround are works well.
```yml
properties:
  nullable_objec:
    oneOf:
      - type: string
        nullable: true
        enum:
          - null
      - $ref: "./object.yml"
```